packages/shopify-codemod/test/transforms/simplify-?=-if-else-statement.test.js

```diff
@@ -0,0 +1,8 @@
+import 'test-helper';
```

Comment:
My main reason for carping about this is that I forget about special symbols when I'm grepping for filenames.  

My incompetence is your inconvenience 👺